src/app/how-council-works/page.tsx

```diff
@@ -256,6 +302,26 @@ export default function HowCouncilWorks() {
             </span>
           </SectionText>
         </Section>
+        <Section>
+          <SectionHeading>Stay Informed, Take Action</SectionHeading>
+          <SectionText>
+            Now that you know how City Council works, why not take the next
+            step? Stay informed and make your voice heard! Check out our{' '}
+            <a href={menuItems[3].href} className="classic-link">
```

Comment:
I think aray-indexes into the nav end up being kinda fragile, since someone might reorder the array and then the link goes to something unrelated
(also we could use the `<Link` component here)
```suggestion
            <Link href="/actions" className="classic-link">
```